lib/std/c.zig

```diff
@@ -1426,7 +1428,7 @@ pub const NAME_MAX = switch (native_os) {
 pub const PATH_MAX = switch (native_os) {
     .linux => linux.PATH_MAX,
     .emscripten => emscripten.PATH_MAX,
-    .wasi => 4096,
+    .wasi => std.fs.wasi.PATH_MAX,
```

Comment:
libc abstraction should not reach into `std.fs` for anything.

Reply:
Removed all std.fs references from c.zig.